specs/wp-signup-spec.js

```diff
@@ -271,96 +229,89 @@ testDescribe( `[${host}] Sign Up  (${screenSize}, ${locale})`, function() {
-					this.signupStepComponent = new SignupStepComponent( driver );
-					this.signupStepComponent.waitForSignupStepLoad();
-					return this.findADomainComponent.displayed().then( ( displayed ) => {
+					const findADomainComponent = new FindADomainComponent( driver );
+					const signupStepComponent = new SignupStepComponent( driver );
+					signupStepComponent.waitForSignupStepLoad();
+					return findADomainComponent.displayed().then( ( displayed ) => {
 						eyesHelper.eyesScreenshot( driver, eyes, 'Domains Page' );
 						return assert.equal( displayed, true, 'The choose a domain page is not displayed' );
 					} );
 				} );
 
 				test.it( 'Can search for a blog name, can see and select a free WordPress.com blog address in results', function() {
-					this.findADomainComponent.searchForBlogNameAndWaitForResults( blogName );
-					this.findADomainComponent.checkAndRetryForFreeBlogAddresses( expectedBlogAddresses, blogName );
-					this.findADomainComponent.freeBlogAddress().then( ( actualAddress ) => {
+					const findADomainComponent = new FindADomainComponent( driver );
+					findADomainComponent.searchForBlogNameAndWaitForResults( blogName );
+					findADomainComponent.checkAndRetryForFreeBlogAddresses( expectedBlogAddresses, blogName );
+					findADomainComponent.freeBlogAddress().then( ( actualAddress ) => {
 						assert( expectedBlogAddresses.indexOf( actualAddress ) > -1, `The displayed free blog address: '${actualAddress}' was not the expected addresses: '${expectedBlogAddresses}'` );
 						selectedBlogAddress = actualAddress;
 					} );
 
 					eyesHelper.eyesScreenshot( driver, eyes, 'Domains Page Site Address Search' );
-					return this.findADomainComponent.selectFreeAddress();
-				} );
-
-				test.it( 'Verify OAuth error not present', function() {
-					const self = this;
-					return driverHelper.getErrorMessageIfPresent( driver ).then( ( errorMsg ) => {
-						if ( errorMsg !== undefined ) {
-							SlackNotifier.warn( `WARNING: Error message [${errorMsg}] encountered on Find Domain page!` );
-							return self.findADomainComponent.selectFreeAddress();
-						}
-					} );
+					return findADomainComponent.selectFreeAddress();
 				} );
 
 				test.describe( `Step ${stepNum}: Plans`, function() {
 					stepNum++;
 
 					test.it( 'Can then see the plans page', function() {
-						this.pickAPlanPage = new PickAPlanPage( driver );
-						return this.pickAPlanPage.displayed().then( ( displayed ) => {
+						const pickAPlanPage = new PickAPlanPage( driver );
+						return pickAPlanPage.displayed().then( ( displayed ) => {
 							eyesHelper.eyesScreenshot( driver, eyes, 'Plans Page' );
 							return assert.equal( displayed, true, 'The pick a plan page is not displayed' );
 						} );
 					} );
 
 					test.it( 'Can select the premium plan', function() {
-						return this.pickAPlanPage.selectPremiumPlan();
+						const pickAPlanPage = new PickAPlanPage( driver );
+						return pickAPlanPage.selectPremiumPlan();
```

Comment:
Can we combine this step and step above? This should become a bit shorter

Reply:
Good idea - thanks